types/string-replace-webpack-plugin/string-replace-webpack-plugin-tests.ts

```diff
@@ -0,0 +1,17 @@
+import * as StringReplacePlugin from 'string-replace-webpack-plugin';
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
Thanks. Updated